src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1206,6 +1207,30 @@ private String stripIndexPhpOrAppsFiles(String url, EditText mHostUrlInput) {
         return url;
     }
 
+
+    private String subdomainToLower(String url, EditText mHostUrlInput) {
+        if(url.toLowerCase().startsWith("http://") ||
+                url.toLowerCase().startsWith("https://")) {
+            if(url.indexOf("/", 8) != -1)
+                url = url.substring(0, url.indexOf("/", 8)).toLowerCase() + url.substring(url.indexOf("/", 8), url.length());

```

Comment:
Please, wrap lines at 120 characters so that they can be read in Github in a glance, without using the horizontal scrollbar.